cmake/Macros.cmake

```diff
@@ -265,6 +265,42 @@ macro(sfml_add_example target)
 
 endmacro()
 
+# add a new target which is a SFML test
+# example: sfml_add_test(sfml-test
+#                           ftp.cpp ...
+#                           sfml-network)
+function(sfml_add_test target SOURCES DEPENDS)
+
+    # set a source group for the source files
+    source_group("" FILES ${SOURCES})
+
+    # check whether resources must be added in target
+    set(target_input ${SOURCES})
```

Comment:
Me neither. 🤔 
It just sets `target_input` and then uses that further down instead of the `${SOURCES}` but it doesn't get used any further. Maybe some leftover, @JonnyPtn?

Reply:
Oh, I just checked the history and noticed by chance that this is just an copy-paste artifact. The example macro adds resource files to that target if they are needed. Might be useful at one point, but not right now.